test/SILGen/default_arguments_generic.swift

```diff
@@ -41,6 +41,7 @@ struct InitializableImpl: Initializable {
 // CHECK-LABEL: sil hidden @$S25default_arguments_generic17testInitializableyyF
 func testInitializable() {
   // The ".init" is required to trigger the crash that used to happen.
+  _ = Generic<InitializableImpl>()
```

Comment:
This changes the test case, though…the CHECK lines might match the "wrong" init.

Reply:
Fix coming up.